Automattic/jetpack
# VideoPress: Add uploaded video count to dashboard

Closes #26360

#### Changes proposed in this Pull Request:

* Adds `getUploadedVideoCount` resolver to check if the user has uploaded videos regardless of query
* Adds `isFetching` and `isFetchingUploadedVideoCount` flags to `useVideos`

#### Other information:


#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Go to the VideoPress dashboard with at least one uploaded video
* Search for something that does not return any result and wait for the request to load
* Check that the `Let's add your first video` message does not appear anymore

TODO: Loading state UI.

https://user-images.githubusercontent.com/8486249/191866122-0aba6f6f-ebfb-4539-a281-9fa2cb10ecab.mp4
